src/include/cursor.i

```diff
@@ -198,11 +196,17 @@ __cursor_reset(WT_CURSOR_BTREE *cbt)
 
     /* If the cursor was active, deactivate it. */
     if (F_ISSET(cbt, WT_CBT_ACTIVE)) {
-        if (!F_ISSET(cbt, WT_CBT_NO_TXN))
+        if (!F_ISSET(cbt, WT_CBT_NO_TRACKING))
             __cursor_leave(session);
         F_CLR(cbt, WT_CBT_ACTIVE);
     }
 
+    /* When the count of active cursors in the session goes to zero, there are no
+    * active cursors, and we can release any snapshot we're holding for read committed isolation.
+    */
```

Comment:
The comment is good, but the style is wrong. There should be a line with only `/*` at the start, followed by the rest of the comment. e.g:

```
/*
 * When the count of active cursors in the session goes to zero, there are no
 * active cursors, and we can release any snapshot we're holding for read committed isolation.
 */
```

Reply:
Fixed the comments.